src/client/opamCommands.ml

```diff
@@ -705,55 +704,57 @@ let show cli =
       Arg.(list string) []
   in
   let show_empty =
-    mk_flag ["empty-fields"]
+    mk_flag ~cli cli_original ["empty-fields"]
       "Show fields that are empty. This is implied when $(b,--field) is \
        given."
   in
   let raw =
-    mk_flag ["raw"] "Print the raw opam file for this package" in
+    mk_flag ~cli cli_original ["raw"] "Print the raw opam file for this package" in
   let where =
-    mk_flag ["where"]
+    mk_flag ~cli cli_original ["where"]
       "Print the location of the opam file used for this package" in
   let list_files =
-    mk_flag ["list-files"]
+    mk_flag ~cli cli_original ["list-files"]
       "List the files installed by the package. Equivalent to \
        $(b,--field=installed-files), and only available for installed \
        packages"
   in
   let file =
-    mk_opt ["file"] "FILE" "DEPRECATED: see $(i,--just-file)"
-      Arg.(some existing_filename_or_dash) None
+    mk_opt ~cli (cli_between cli2_0 cli2_1 ~replaced:"--just-file")
+      ["file"] "FILE" ""
```

Comment:
```suggestion
      ["file"] "FILE" "DEPRECATED: use an explicit path argument as package instead. \
              Get package information from the given FILE instead of from \
              known packages. This implies $(b,--raw) unless $(b,--fields) is \
              used. Only raw opam-file fields can be queried."
```
(lifted out of #3729)

Reply:
```suggestion
      ["file"] "FILE" "Get package information from the given FILE instead of from \
              known packages. This implies $(b,--raw) unless $(b,--fields) is \
              used. Only raw opam-file fields can be queried."
```

Reply:
The deprecation/removal notice will be printed by cli versioning mechanism.  Do we need another deprecated notice in its doc ?

Reply:
The message I'd put in there is the one displayed for `OPAMCLI=2.0` which simply matches what opam 2.0 displays at the moment

Reply:
oh ok